Set/Game/GameView.swift

```diff
@@ -66,6 +55,68 @@ struct GameView: View {
 //            }
 //            .padding()
         }
+        .navigationTitle("\(gameViewModel.timerTitle)")
+        .navigationBarTitleDisplayMode(.inline)
+        .navigationBarBackButtonHidden(true)
+        .navigationBarItems(
+            leading: Button {
+                gameViewModel.dealThreeMoreCards()
+            } label: {
+                threeMoreCardsButton
+            }
+            .disabled(gameViewModel.isMoreCardAvailable),
+
+            trailing: Button {
+                gameViewModel.startNewGame()
+            } label: {
+                newGameButton
+            }
+        )
+    }
+
+    private var gameBlock: some View {
+        AspectVGrid(items: gameViewModel.cardsOnScreen, aspectRatio: 2 / 3) { card in
+            cardViewBuilder.build(
+                for: card,
+                isColorBlindModeEnabled: gameViewModel.isColorBlindModeEnabled
+            )
+            .matchedGeometryEffect(id: card.id, in: dealingNamespace)
+            .onTapGesture {
+                gameViewModel.didSelect(card: card)
+            }
+        }
+        .onAppear {
+            for i in 0 ..< 12 {
+                withAnimation(dealAnimation(for: gameViewModel.deck[i])) {
+                    gameViewModel.deal(card: gameViewModel.deck[i])
+                }
+            }
+        }
+    }
+
+    private var discardPileBlock: some View {
```

Comment:
I see that you finish all `View`s with `Block`, why just don't name it like this:
 `discardPileBlock` -> `discardPileView`
  `gameBlock` -> `gameView` and may be find another naming that would describe it more precisely.

Reply:
fixed